scripts/sentiment_analysis/finetune_lm.py

```diff
@@ -94,15 +130,17 @@
     context = mx.cpu()
 else:
     print('Use gpu%d' % args.gpu)
-    context = mx.gpu(args.gpu)
+    context = mx.gpu(int(args.gpu))
```

Comment:
The int cast is not required because `type=int` is already specified in argparse.